### PR TITLE
Update Exchange-Teams-interact.md

### PR DESCRIPTION
**Pull request by carolynblanding**
@dstrome 

David it's endless :) I had previously made change to the table for profile photo support and I guess missed the paragraph at the start that states ability to change photo (or that was added after the fact). We no longer support changing profile photo because method changed from MT EWS call to Graph Explorer call which requires REST API support for the Exchange Mailbox. From my testing I find that a 404 is returned when attempting to save the photo and user gets and error message stating that there was a problem saving the photo please try again. We require some work from engineering here too because we shouldn't show an option to change if not supported. For now, I wanted to get this article consistent with the current experience and have the table and text align. 

Do you know who owns this support topic which is also now wrong on this point: https://support.microsoft.com/en-us/office/change-your-picture-in-teams-7a711943-9248-420e-b814-c071aa8d9b9c#ID0EAABAAA=Desktop

### CODE REVIEW

**Comment by carolynblanding** [comment]
@claudialake Claudia, Joao Loureiro suggested I loop you in on this PR request. Change from MT EWS to Graph Explorer using REST API's has broken photo upload for Exchange on-premises users. I had updated the table previously but there was also a sentence added to the doc which said on-premises users could change their photo. We also have a conflict with this doc https://support.microsoft.com/en-us/office/change-your-picture-in-teams-7a711943-9248-420e-b814-c071aa8d9b9c#ID0EAABAAA=Desktop

The Teams client isn't behaving correctly either because it lets users start the change photo process. They can upload a new photo but when they select to change it fails with a message to please retry. The client should provide the right feedback that it's not supported. 

Do you know if REST API will be supported for Exchange on-premises mailbox?

**Comment by claudialake** [comment]
> @claudialake Claudia, Joao Loureiro suggested I loop you in on this PR request. Change from MT EWS to Graph Explorer using REST API's has broken photo upload for Exchange on-premises users. I had updated the table previously but there was also a sentence added to the doc which said on-premises users could change their photo. We also have a conflict with this doc https://support.microsoft.com/en-us/office/change-your-picture-in-teams-7a711943-9248-420e-b814-c071aa8d9b9c#ID0EAABAAA=Desktop
> 
> The Teams client isn't behaving correctly either because it lets users start the change photo process. They can upload a new photo but when they select to change it fails with a message to please retry. The client should provide the right feedback that it's not supported.
> 
> Do you know if REST API will be supported for Exchange on-premises mailbox?

I'm sorry, I don't know the answer. I am the Content PM for the Teams troubleshooting content. You might check with someone on  the Exchange side.

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/Exchange-Teams-interact.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Users hosted on Exchange Online or Exchange Dedicated vNext can use all the feat
...
+Users with mailboxes hosted on-premises must be synchronized to Azure Active Directory. They can make use of all the features in the above scenario, but additionally they can  manage meetings, providing Exchange Server 2016 (Cumulative Update 3), or later, is running on-premises with OAuth configured (preferably via the Exchange Hybrid Configuration Wizard) as described in [Configure OAuth authentication between Exchange and Exchange Online organizations](https://docs.microsoft.com/exchange/configure-oauth-authentication-between-exchange-and-exchange-online-organizations-exchange-2013-help). To enable calendar delegation for these users, you must also complete steps 2-3 as described in [Configure Integration and OAuth between Skype for Business Online and Exchange Server](https://docs.microsoft.com/skypeforbusiness/deploy/integrate-with-exchange-server/oauth-with-online-and-on-premises); these steps will provide the Teams scheduling application the required permissions to confirm delegate permissions.
```
Comment (JohanFreelancer9) [inline Teams/Exchange-Teams-interact.md:34]:
```suggestion
Users with mailboxes hosted on-premises must be synchronized to Azure Active Directory. They can make use of all the features in the above scenario, but additionally they can manage meetings, providing Exchange Server 2016 (Cumulative Update 3), or later, is running on-premises with OAuth configured (preferably via the Exchange Hybrid Configuration Wizard) as described in [Configure OAuth authentication between Exchange and Exchange Online organizations](https://docs.microsoft.com/exchange/configure-oauth-authentication-between-exchange-and-exchange-online-organizations-exchange-2013-help). To enable calendar delegation for these users, you must also complete steps 2-3 as described in [Configure Integration and OAuth between Skype for Business Online and Exchange Server](https://docs.microsoft.com/skypeforbusiness/deploy/integrate-with-exchange-server/oauth-with-online-and-on-premises); these steps will provide the Teams scheduling application the required permissions to confirm delegate permissions.
```

Comment (JohanFreelancer9) [reply Teams/Exchange-Teams-interact.md:34]: Please remove extra space between the words "can" and "manage" at the end of the second line.

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dstrome** [comment]
@Cloud-Writer - Can you help @carolynblanding update the SMC article referenced in the original PR comment above?

**Pull request merged**